kmamiz_data_processor/.env

```
@@ -0,0 +1,7 @@
+BIND_IP=0.0.0.0
+PORT=8080
```

```suggestion
PORT=8000
```
 Same as above, change to `8000` to avoid collision with K8s proxy.